vendor/github.com/ciscoecosystem/dcnm-go-client/client/client.go

```diff
@@ -337,20 +338,30 @@ func (c *Client) authenticate() error {
 func (c *Client) Do(req *http.Request) (*container.Container, *http.Response, error) {
 	log.Println("[DEBUG] Begining Do method ", req.URL.String())
 
+	reqDump, err := httputil.DumpRequestOut(req, true)
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	resp, err := c.httpClient.Do(req)
 	if err != nil {
 		return nil, nil, err
 	}
-	log.Println("[DEBUG] HTTP Request ", req.Method, req.URL.String())
-	log.Println("[DEBUG] HTTP Response ", resp.StatusCode, resp)
+
+	respDump, err := httputil.DumpResponse(resp, true)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Printf("[DEBUG] \n--[ HTTP Request ]------------------------------------ \n %s\n---------------------------------------------\n", string(reqDump))
+	log.Printf("[DEBUG] \n--[ HTTP Response ]----------------------------------- \n %s\n---------------------------------------------\n", string(respDump))
```

Comment:
This will print to the log all HTTP Request including the login / authentication payload. Can we avoid that?

Reply:
The ACI go client uses a variable (skipPayload...) to avoid printing the payload when using the login/authentication method.